src/AssetsTransferApi.ts

```diff
@@ -156,6 +160,44 @@ export class AssetsTransferApi {
 			opts?.format
 		);
 	}
+	/**
+	 * Fetch estimated fee information for an extrinsic
+	 *
+	 * @param tx TxResult<T>
+	 */
+	public async fetchFeeInfo<T extends Format>(
+		tx: SubmittableExtrinsic<'promise', ISubmittableResult>,
+		format?: T
+	): Promise<RuntimeDispatchInfo | RuntimeDispatchInfoV1 | null> {
+		const { _api: api } = this;
```

Comment:
Should we keep this in line with the rest of the API where its `_api` to denote its read only, if you think its easier to read like `api` then I am happy to have a follow up PR change the rest of the API.

Reply:
totally. Ill change this to `_api`